service/src/test/java/bio/terra/workspace/pact/ProfileApiTest.java

```diff
@@ -41,6 +41,7 @@ public RequestResponsePact existingAzureBillingProfile(PactDslWithProvider build
             .stringType("managedResourceGroupId")
             .uuid("id");
     return builder
+        .usingLegacyDsl()
```

Comment:
`usingLegacyDsl()`, `PactBuilder` and returning `V4Pact` are all weird artifacts which _I believe_ are transitional and may be able to go away if we bump to an _even later_ version of Pact (as we have done in WDS).  However, WSM is currently using a version of `gradle` which is incompatible with the newer versions of Pact JVM and so here they remain.

Reply:
Oops, I guess I didn't need to do this at all.  Turns out, we just needed to explicitly specify `PactVersion` as a param to avoid this.  Unrolled this in 70403afc97c79a52e3a3bb81263fed1a2cd8f1d3